cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -29,6 +29,7 @@ type MasterConfiguration struct {
 	Networking        Networking `json:"networking"`
 	KubernetesVersion string     `json:"kubernetesVersion"`
 	CloudProvider     string     `json:"cloudProvider"`
+	AuthorizationMode string     `json:"authorizationMode"`
```

Comment:
Is there anything that blocks us from defaulting to RBAC now already? If not, please set RBAC to the default authz mode in defaults.go